code-challenges.test.js

```diff
@@ -13,44 +13,187 @@
+// // The .filter method will be used to filter through the values offullArr1 and fullArr2.
+// // The typeof command will be used to find number values
+// // Odd numbers will be outputed by setting up a modulo that determines which number are not equal to 0
+// // The .sort method will be used on oddNums
 
-const numbersToAdd2 = [0, 7, -8, 12]
-// Expected output: [0, 7, -1, 11]
+const sortOdd = (array) => {
+let oddNums = array.filter((value)=> typeof value === 'number' && value % 2 !==0)
+  return oddNums.sort((a,b) => a-b)
+}
+console.log(sortOdd(fullArr1))
+console.log(sortOdd(fullArr2))
 
-const numbersToAdd3 = []
-// Expected output: []
+// PASS  ./code-challenges.test.js
+// ● Console
 
+//   console.log
+//     [ -9, 7, 9, 199 ]
 
-// b) Create the function that makes the test pass.
+//     at Object.log (code-challenges.test.js:110:9)
+
+//   console.log
+//     [ -823, 7, 23 ]
+
+//     at Object.log (code-challenges.test.js:111:9)
+
+    //* Received test suite failed error but received a pass on my output
+
+
+// // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+
+// // a) Create a test with expect statements for each of the variables provided.
+
+var numbersToAdd1 = [2, 4, 45, 9]
+// Expected output: [2, 6, 51, 60]
+var numbersToAdd2 = [0, 7, -8, 12]
+// Expected output: [0, 7, -1, 11]
+var numbersToAdd3 = []
+// Expected output: []
+
+describe('aSum', () => {
+  it("takes in an array, returns an array of the accumulating sum", () => {
+    expect(aSum(numbersToAdd1)).toEqual([2, 6, 51, 60]);
+    expect(aSum(numbersToAdd2)).toEqual([0, 7, -1, 11]);
+    expect(aSum(numbersToAdd3)).toEqual([]);
+  })
+})
+
+  // FAIL  ./code-challenges.test.js
+  // ● aSum › takes in an array, returns an array of the accumulating sum
+
+  //   ReferenceError: aSum is not defined
+
+// // b) Create the function that makes the test pass.
+
+//PsuedoCode:
+// A function will be created called aSum with array declared as the value
+// A for loop will be used to itterate over an array with the output being the sum of the array
+//Console log will be used to display 'numbersToAdd' variable
+
+const aSum = (array) => {
+  for(i = 1; i <array.length; i++ ){
+  array[i] = array[i] + array[i - 1]
+}
+  return array
+}
+
+console.log(aSum(numbersToAdd1));
+console.log(aSum(numbersToAdd2));
+console.log(aSum(numbersToAdd3));
+
+//* I have been having trouble getting this one to pass. Below is the error message I receive. I have tried rearranging the syntax in the for loop but have not been able to get it to pass.
+
+// ● aSum › takes in an array, returns an array of the accumulating sum
+
+// expect(received).toEqual(expected) // deep equality
+
+// - Expected  - 3
+// + Received  + 3
+
+//   Array [
+//     2,
+// -   6,
+// -   51,
+// -   60,
+// +   8,
+// +   59,
+// +   119,
+//   ]
+
+//   143 | describe('aSum', () => {
```

Comment:
not sure about the problem. It works on my side. i copied and pasted it on vs code